testdata/baselines/reference/submodule/compiler/unusedImports1.errors.txt.diff

```diff
@@ -0,0 +1,15 @@
+--- old.unusedImports1.errors.txt
++++ new.unusedImports1.errors.txt
+@@= skipped -0, +0 lines =@@
+-file2.ts(1,1): error TS6133: 'Calculator' is declared but its value is never read.
++file2.ts(1,9): error TS6133: 'Calculator' is declared but its value is never read.
+ 
+ 
+ ==== file1.ts (0 errors) ====
+@@= skipped -7, +7 lines =@@
+     
+ ==== file2.ts (1 errors) ====
+     import {Calculator} from "./file1"
+-    ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
```

Comment:
Hm, the old range here was the entire import too...

Reply:
Yeah, the old compiler's logic is just kinda wonky and incomplete. Doesn't make sense to talk about "all" when there is only one, and doesn't make sense to highlight a whole statement when you can just highlight the actual identifier.

Reply:
That said, I think backporting is a waste of time. Better to just accept the changes and move forward.

Reply:
I don't entirely agree; it'd be nice to get these changes out to people and fix up our baselines, if it's easy to do so.